nullaway/src/test/java/com/uber/nullaway/NullAwayCoreTests.java

```diff
@@ -623,6 +623,29 @@ public void nullableOnJavaLangVoid() {
         .doTest();
   }
 
+  @Test
+  public void nullableOnJavaLangVoidWithCast() {
+    defaultCompilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  Void foo1() {",
+            "    // temporarily, we treat a Void return type as if it was @Nullable Void",
+            "    return (Void)null;",
+            "  }",
+            "  void consumeVoid(Void v) {",
```

Comment:
Can we add a comment here indicating the formal is treated as if it were `@Nullable`? I think having the comment just at the call site confused me a bit